core/src/main/java/io/confluent/kafka/schemaregistry/storage/KafkaStore.java

```diff
@@ -182,12 +174,40 @@ private void createSchemaTopic() throws StoreInitializationException {
     schemaTopicProps.put(LogConfig.CleanupPolicyProp(), "compact");
 
     try {
-      AdminUtils.createTopic(zkUtils, topic, 1, schemaTopicReplicationFactor, schemaTopicProps);
+      AdminUtils.createTopic(zkUtils, topic, 1, schemaTopicReplicationFactor, schemaTopicProps,
+              RackAwareMode.Enforced$.MODULE$);
     } catch (TopicExistsException e) {
       // This is ok.
     }
   }
 
+  private String getBrokerEndpointsFromZk(Seq<Broker> brokers) {
```

Comment:
Might be good to have a unit test for this? 

Also, I think we've seen from some users when they have some difficulties with setup that it would be useful to log the list of endpoints collected from zookeeper.